core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -97,4 +97,7 @@ private TableProperties() {}
 
   public static final String WRITE_AUDIT_PUBLISH_ENABLED = "write.wap.enabled";
   public static final String WRITE_AUDIT_PUBLISH_ENABLED_DEFAULT = "false";
+
+  public static final String HIVE_CHECK_LOCK_STATE_WAIT_MS = "hive.lock.check-state-wait-ms";
```

Comment:
I am not sure if it should be a table property. Also, I am open to any naming suggestions.

Reply:
nit: I wonder if there's a better place to put this configs, maybe in HiveTableOperations?

Reply:
Yeah, same for me. Maybe, we can set it in Hadoop config using some certain prefix.

Reply:
If we aren't sure, then let's remove the property. We don't want to add a property that we'll have to remove support for later if we can avoid it.

Reply:
@rdblue are you suggesting to hardcode the wait time? Table props is not the right place, which is clear. The question whether we want to have that property at all.

Reply:
Yeah, we can always add a configuration setting later if we want one. For now though, we definitely need this and don't think table setting is the right place for it. So let's decide later.

Reply:
Done